pywbem/_utils.py

```diff
@@ -97,6 +97,65 @@ def _to__bytes(obj):
     return obj.encode("utf-8")
 
 
+def _eq_name(name1, name2):
+    """
+    Test two CIM names for equality.
+
+    The comparison is performed case-insensitively.
+
+    One or both of the names may be `None`; Two names that are `None` are
+    considered equal.
+    """
+    if name1 is None:
+        if name2 is None:
```

Comment:
Would this be any faster if these 4 lines were one return false if None else True.   .??

Reply:
It ended up being a little faster. See updated initial comment.